lib/galaxy/webapps/base/controller.py

```diff
@@ -276,6 +276,10 @@ def client(self, trans, **kwd):
         return self._bootstrapped_client(trans, **kwd)
 
     def _bootstrapped_client(self, trans, app_name='analysis', **kwd):
+        # This includes contextualized user options in the bootstrapped data; we don't want to cache it.
+        trans.response.headers['Cache-Control'] = ['no-cache', 'no-store', 'must-revalidate']
```

Comment:
Might make sense to consolidate this in a decorator (in dev) ? We do this in a couple of places.

Reply:
Yep, I actually started doing exactly that yesterday, but for a separate PR for dev